### DISCUSSION TITLE
Mixcore_v1.0.1 Installation couldn't pass database entry model window

### DISCUSSION THREAD

**Discussion by dush-a** [mixcore/mix.core#379] [Q&A] [answered]
I tried to install Mixcore_v1.0.1 in Plesk and then local IIS server in Windows 10, in both occasions it couldn't pass the first model window with database configuration section. I am using a named instance of MSSQL server 2017 Developer edition with mix mode for local IIS installation. (instance name SQL2017DEV). I have already created  SQL Login account, Database and Database user for this installation.

When I click connect it trying for few seconds and pops up red error message at the bottom of the screen with a red bell icon. The message is just "Server Errors"

I tried following for the Server name IP field:
.\SQL2017DEV
myPC1\SQL2017DEV
SQL2017DEV

What am I doing wrong?
Is there any sample values for those fields?
Is there a way to find the detailed error message ?
Thanks

**Comment by Smilefounder** [comment]
Hi @dush-a , is there any screenshots that you can share for easier to see the issue? Or could you share with me the URL to access?

Btw, are you able to access your DB via SQL Server Management Studio (SSMS)?

Thanks.

**Reply by dush-a** [reply]
Hi @Smilefounder , thanks for the comment. I managed to get it to work in the local Windows 10 IIS. I have enabled dbowner, datareader, datawriter for the user  located inside security/users folder found under the database. But it didn't worked, Then I did the same for the user under Security/Login folder. I think that fixed the issue.

I have given "myPC1\SQL2017Dev" for the Server name IP field.
By the way, so far the app is great!. Good work!

**Reply by Smilefounder** [reply]
thanks @dush-a for letting us know!

**Comment by dush-a** [comment, marked as answer]
I have enabled dbowner, datareader, datawriter for the user located inside security/users folder found under the database. But it didn't worked, Then I did the same for the user under Security/Login folder. I think that fixed the issue.

I have given "myPC1\SQL2017Dev" for the Server name IP field.

**Reply by alikazai** [reply]
this worked for me thanks
